test/common/ssl/test_data/certs.sh

```diff
@@ -68,10 +69,24 @@ openssl req -new -x509 -days 730 -key selfsigned_key.pem -out selfsigned_cert.pe
 openssl req -new -key expired_key.pem -out expired_cert.csr -config selfsigned_cert.cfg -batch -sha256
 openssl x509 -req -days -365 -in expired_cert.csr -signkey expired_key.pem -out expired_cert.pem
 
+# Generate revoked_cert.pem
```

Comment:
We could simply revoke one of the existing certs and add it to the CRL.

If you prefer `revoked_cert.pem`, that's fine with me, but perhaps reuse `san_with_dns.cfg` to produce cert instead of adding another copy of the same `.cfg` file?

Reply:
Good point - I just pushed a commit that revokes an existing cert and tweaks the tests.